data/faq/steal_project_idea.md

```diff
@@ -0,0 +1,9 @@
+The whole point of "Open Source Software" is to encourage "stealing".
```

Comment:
That 'stealing' part was just a symbolism, lets take it in right taste? sth like 
```
The whole point of "Open Source Software" is to encourage taking code from places, using and improving it.
```
```
The whole point of "Open Source Software" is to encourage freedom to use any code in the world, without obligations
```
```
The whole point of "Open Source Software" is to encourage celebration of complete independence in sharing and writing code
```